rust/agama-cli/src/lib.rs

```diff
@@ -211,6 +232,10 @@ pub async fn run_command(cli: Cli) -> Result<(), ServiceError> {
             let manager = build_manager().await?;
             install(&manager, 3).await?
         }
+        Commands::Finish { method } => {
+            let manager = build_manager().await?;
+            finish(&manager, method).await?;
```

Comment:
You can do the `method` argument as mandatory and then use the `unwrap_or_default` function.